src/versions.js

```diff
@@ -36,7 +63,17 @@ function getVersion( resource, version ) { // jshint ignore:line
 	if ( version === undefined ) {
 		return resource;
 	} else {
-		return getVersions( resource )[ version ] || resource;
+		var versions = getVersions( resource );
+		version = parseInt( version );
```

Comment:
Calvin pointed out the `radix` parameter. Apparently the [default behavior](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) will treat the value as octal if it starts with a "0". :tired_face: